css/properties.json

```diff
@@ -120,6 +120,21 @@
     "order": "uniqueOrder",
     "status": "nonstandard"
   },
+  "-moz-context-properties": {
+    "syntax": "fill || stroke",
```

Comment:
Yep, see https://www.w3.org/TR/css3-values/
`fill | stroke` - enum
`[]#` - group occurs one or more times, separated by comma

Correction: property can contain `none` value, so syntax should be `none | [ fill | stroke]#`